features/api/application_plans.feature

```diff
@@ -50,6 +50,11 @@ Feature: Application plans index page
     Then the plan is published
     And a buyer will be able to use it for their applications
 
+  Scenario: Unset the default application plan
```

Comment:
May I suggest moving this up right after _Set a default application plan_ 🤓

Reply:
Done